code/modules/client/verbs/ooc.dm

```diff
@@ -23,7 +23,10 @@
 			to_chat(src, "<span class='danger'>You have been banned from OOC.</span>")
 			return
 
-	msg = utf8_sanitize(msg, src, MAX_MESSAGE_LEN)
+	if(CONFIG_GET(flag/use_libvg))
+		msg = utf8_sanitize(msg, src, MAX_MESSAGE_LEN)
+	else
+		msg = copytext(sanitize(msg), 1, MAX_MESSAGE_LEN)
```

Comment:
it would be less work in the long run if you just made the utf8 procs redirect the non-utf if use libvg is disabled.

this way we can expand more shit to use those procs without having to add a fuck ton of checks everywhere